tools/editor/filesystem_dock.cpp

```diff
@@ -38,14 +38,16 @@
 #include "editor_settings.h"
 #include "scene/main/viewport.h"
 
-
 bool FileSystemDock::_create_tree(TreeItem *p_parent,EditorFileSystemDirectory *p_dir) {
 
 
 	TreeItem *item = tree->create_item(p_parent);
 	String dname=p_dir->get_name();
-	if (dname=="")
+	if (dname=="") 
 		dname="res://";
+	else
+		// collapse every tree item but the root folder
+		item->set_collapsed(true);
```

Comment:
Please use braces for this `else`, even if the comment is properly ignored by the compiler and the statement is indeed part of the `else` block, this looks quite confusing.